src/pages/Withdraw.tsx

```diff
@@ -35,8 +41,21 @@ function Withdraw({ poolName }: Props): ReactElement {
   const swapContract = useSwapContract(poolName)
   const { account } = useActiveWeb3React()
   const POOL = POOLS_MAP[poolName]
+  const [withdrawLPTokenAmount, setWithdrawLPTokenAmount] =
+    useState<BigNumber>(Zero)
+
+  const tokenInputSum = useMemo(
+    () =>
+      POOL.poolTokens.reduce(
+        (sum, { symbol }) =>
+          sum.add(
+            parseEther(withdrawFormState.tokenInputs[symbol].valueRaw) || Zero,
```

Comment:
Does this handle parsing "" into a bignum?

Reply:
Ohhh!
It makes error.
Does it bug of ether.js? or is there any reason they do this?

Reply:
@hammeiam 

I have fixed like the following;

```
           parseEther(
              withdrawFormState.tokenInputs[symbol].valueRaw.trim() || "0",
            ) || Zero,
```

Reply:
I think your solution of doing || "0" should work. I don't believe you'll need || Zero after it since parseEther will never evaluate to a falsey value

Reply:
@hammeiam  Before merge I didn't check your message. You are right! we can remove ||Zero.
Let me remove.